Assets/GravitationalWaveSurfer/Source/GWS/Environment/Runtime/ChunkManager.cs

```diff
@@ -183,6 +183,8 @@ public void UpdateChunks()
         /// <param name="chunkPos">chunk coordinate of new chunk</param>
         private void GenerateChunk(Vector3Int chunkPos, bool awake=false)
         {
+            if (!GWManager.Instance) return; 
```

Comment:
did you encounter a problem where GenerateChunk() is called before GWManager.awake()?

Reply:
No I just couldn't use the chunk spawning without a GWManager because how coupled they were otherwise it worked fine!